extensions/interactions/LabelingInput/LabelingInput.html

```diff
@@ -0,0 +1,121 @@
+<script src="{{cache_slug}}/extensions/interactions/LabelingInput/LabelingInput.js"></script>
+<script src="{{cache_slug}}/extensions/interactions/LabelingInput/validator.js"></script>
+
+<script type="text/ng-template" id="interaction/LabelingInput">
+  <style>
+    .image-click-container {
+      background: #fff;
+      padding: 8px;
+      position: relative;
+    }
+
+    .oppia-image-click-img {
+      cursor: pointer;
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+    }
+    @media screen and (max-width: 959px) {
+      .code-repl-container {
+        width: 100vw;
+      }
+      .oppia-image-click-img {
+        max-width: 100vw;
+      }
+      .image-click-container {
+        padding: 8px 0px;
+      }
+    }
+  </style>
+  <div ng-if="!imageUrl" translate="I18N_INTERACTIONS_IMAGE_CLICK_SELECT"></div>
+
+  <div class="image-click-container" ng-if="imageUrl">
+  <h2><[imageTitle]></h2>
+  <hr>
+  <h4>Word Bank</h4>
+    <div data-drop="true"
+        jqyoui-droppable="{animate:true}">
+      </div>
+      <div ng-repeat="word in regionsAndBonus"
+        style="
```

Comment:
Large styles like this should be moved into classes, and the attributes alphabetized.